habitat-lab/habitat/config/benchmark/rearrange/nav_pick.yaml

```diff
@@ -1,41 +0,0 @@
-# @package _global_
-
```

Comment:
Are you sure we should remove this benchmark?

Reply:
Yes, I think this should be superseded by rearrange-easy.